src/functionalTest/java/uk/gov/hmcts/reform/hmi/helper/RestClientHelper.java

```diff
@@ -82,6 +82,74 @@ public void performPostRequestAndValidate(String payload,
                 .statusCode(expectedStatusCode);
     }
 
+    /**
+     * Perform a post request without an OAuth token and validate the response.
```

Comment:
Update the javadoc for new comments to get as currently comments say its a post request.

Reply:
updated.